app/helpers/riiif_helper.rb

```diff
@@ -0,0 +1,12 @@
+module RiiifHelper
+  # use relative url unless we've defind a riiif server in config/environments/*.rb
+  def riiif_image_url (file_set_id)
+    file_id = FileSet.find(file_set_id).original_file.id
+    path = riiif.info_path(file_id, locale: nil)
+    begin
+      return URI.join(Rails.configuration.riiif_server, path).to_s
+    rescue NoMethodError
+      return path
```

Comment:
what circumstances is this rescuing from, what method ends up missing?  If you don't have `Rails.configuration.riiif_server`?  I think a `Rails.configuration.respond_to?(:riiif_server)` is better than a rescue.